llvm/test/CodeGen/AMDGPU/conditional-mem-no-cbranch-execz.ll

```diff
@@ -0,0 +1,66 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx942 -mattr=+sramecc,-xnack < %s | FileCheck %s
+
```

Comment:
Need some IR pass checks

Reply:
Added tests for the pass in [annotate-likely-varying-branches.ll](https://github.com/llvm/llvm-project/pull/117567/commits/8cf0cbf8fb44c9630cb48f51f3b2e22b68598272#diff-561a228c0e2cb1220a56bc2619f9e0b87407a101372049e9ba68858381255183). I'll update them when changing to a different annotation mechanism instead of branch weights.